python/lsst/meas/extensions/scarlet/scarletDeblendTask.py

```diff
@@ -316,6 +246,89 @@ def deblend(mExposure, footprint, config, spectrumInit):
     return blend, skipped
 
 
+def buildLiteObservation(
+    modelPsf,
+    psfCenter,
+    mExposure,
+    footprint=None,
+    badPixelMasks=None,
+    useWeights=True,
+    convolutionType="real",
+):
+    """Generate a LiteObservation from a set of parameters.
+
+    Make the generation and reconstruction of a scarlet model consistent
+    by building either a `DummyObservation` or `LiteObservation` from a
+    set of parameters. Because this method serves a dual purpose, some
+    parameters (`mExposure`, `footprint`, `badPixelMasks`) are optional
+    when the observation is only used to reconstruct a model,
+    while others (`modelBox`, `observedPsfs`, `dtype`) are automatically
+    generated from `mExposure`, so they are unnecessary when building a
+    full observation.
```

Comment:
If you have two orthogonal sets of parameters that each only apply to one usage, consider making them DataClasses to at least simplify usage and/or docs.